types/storybook__addon-knobs/index.d.ts

```diff
@@ -53,6 +53,7 @@ export function object<T>(name: string, value: T, groupId?: string): T;
 export function radios<T>(name: string, options: { [s: string]: T }, value?: T, groupId?: string): T;
 
 export function select<T>(name: string, options: { [s: string]: T }, value: T, groupId?: string): T;
+export function select<T>(name: string, options: ReadonlyArray<T>, value: T, groupId?: string): T;
```

Comment:
@hipstersmoothie can you link me to the docs describing this interface, please?

Reply:
https://github.com/storybookjs/storybook/tree/master/addons/knobs#select

Reply:
> You can also provide options as an array like this: ['red', 'blue', 'yellow']

Reply:
Yep, saw it in https://github.com/storybookjs/storybook/blob/master/addons/knobs/README.md#select - not sure why I missed that before. Do you mind adding an example in the tests?

Reply:
Sure!

Reply:
@joscha I have added a test that reflects how I use a select the most. Without my change the test fails ✅